### ISSUE TITLE
⌃⌘S no longer works

### ISSUE THREAD

**Issue by wadetregaskis** [Ranchero-Software/NetNewsWire#4262]
Now, in addition to the sidebar _always_ being open at launch even though I always hide it, I have to actually mouse through the menus to hide it, rather than hitting the keyboard shortcut.

I don't know when this broke… maybe a weekish ago?

6.1.4 (6120).

**Comment by brentsimmons** [comment]
The app hasn’t updated in that time. Is there some other change (macOS update, for instance) that might account for this?

**Comment by wadetregaskis** [comment]
Certainly possible, although nothing comes to mind.

**Comment by wadetregaskis** [comment]
I finally figured it out, after seeing that the keyboard shortcut would randomly work on some days but not others.  A few weeks ago I started using [Display Maid](https://funk-isoft.com/display-maid.html), and it turns out it swallows this shortcut for some reason (even though it serves no apparent function in that app).

So this is not a bug in NetNewsWire nor anything NetNewsWire can do anything about, as far as I know.  Sorry for the noise.
